graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGen.kt

```diff
@@ -412,7 +412,8 @@ data class CodeGenConfig(
     val snakeCaseConstantNames: Boolean = false,
     val generateInterfaceSetters: Boolean = true,
     var javaGenerateAllConstructor: Boolean = true,
-    val implementSerializable: Boolean = false
+    val implementSerializable: Boolean = false,
+    val generateGeneratedAnnotation: Boolean = false
```

Comment:
I find the naming of this config confusing. Perhaps rename to addGeneratedAnnotation?

Reply:
Good point. addGeneratedAnnotation is much better.